### PR TITLE
RND-565 Add DDS text fonts

### PR DESCRIPTION
**Pull request by kubama**
## Description
Added DDS text fonts

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [X] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [X] I added proper labels to this PR.

## Tests
N/A

 

## Documentation
Updated READMEs

### CODE REVIEW

File: packages/frontend/styles/font-dds-Roboto.css [hunk 1]
```diff
@@ -0,0 +1,217 @@
+@charset "UTF-8";
+/* The order of the font-face is important. If the woff is last for the font-wieght Webkit broswers
+will end up using the woff file over the unicode-range version. If the woff is first for the font-wieght
+the browser will prefer the unicode-range woff2 version over the woff */
```
Comment (PatrycjaKubiczek) [inline packages/frontend/styles/font-dds-Roboto.css:4]:
```suggestion
/* The order of the font-face is important. If the woff is last for the font-weight Webkit browsers
will end up using the woff file over the unicode-range version. If the woff is first for the font-weight
the browser will prefer the unicode-range woff2 version over the woff */
```

File: packages/frontend/styles/font-dds-Roboto.css [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  src: url("../fonts/dell/Roboto-Medium_g.woff2") format("woff2");
+  unicode-range: U+0370-03FF;
+}
+/* vietnamese */
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url("//gth") format("woff2");
```
Comment (PatrycjaKubiczek) [inline packages/frontend/styles/font-dds-Roboto.css:215]: is this the right URL? 🤔

**Review by qooban — DISMISSED** [review]
We will have duplicated Roboto font, but I assume this is intermediate stage and we will remove the one previously added in the future.

File: packages/frontend/styles/font-dds-Roboto.css (hunk 2 above)
Comment (kubama) [reply packages/frontend/styles/font-dds-Roboto.css]: Apparently not :). Fixed.

**Review by PatrycjaKubiczek — APPROVED** [review]

**Pull request merged**